bugbug/db.py

```diff
@@ -55,10 +63,22 @@ def download_support_file(path, file_name):
         print(f"Downloading {url} to {path}")
         utils.download_check_etag(url, path)
 
-        if path.endswith(".xz"):
-            extract_file(path[:-3])
+        if path.endswith(".zst"):
+            extract_file(path[:-4], "zst")
+
     except requests.exceptions.HTTPError:
-        print(f"{file_name} is not yet available to download for {path}")
+        try:
+            url = re.sub(".zst$", ".xz", url)
+            path = re.sub(".zst$", ".xz", path)
```

Comment:
You need to escape the `.`, or it will match any character.

You could also implement this with os.path.splitext, something like:
```Python
url = f"{os.path.splitext(url)[0]}.xz"
path = f"{os.path.splitext(path)[0]}.xz"
```